.github/workflows/tests.yml

```diff
@@ -0,0 +1,51 @@
+name: tests
+
+on: [push, pull_request]
+
+env:
+  PYTHONHASHSEED: 1042466059
+
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [2.7, pypy2, pypy3, 3.6, 3.7, 3.8, 3.9]
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Pip cache
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('setup.*') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip setuptools wheel
+        python -m pip install -U coverage
+        python -m pip install -U -e ".[test,docs]"
+    - name: Test
+      run: |
+        coverage run -m zope.testrunner --test-path=src  --auto-color --auto-progress
+        coverage run -a -m sphinx -b doctest -d docs/_build/doctrees docs docs/_build/doctests
```

Comment:
In more recent workflows, I add a step after this that does 'coverage report' so we can see what's happening there.